### ISSUE TITLE
question

### ISSUE THREAD

**Issue by by-tech** [Fju/zettelwirtschaft#1]
Hi

we will start using this object detector program for receipt.
but before we have some question : 

1/ the main problem is dataset and i don't think it's possible to have for example 10000 receipts already labeled.
So do you think only 35 images labeled is enouph ?  

2/ how you train data set ?

3/ How you test your model ?

Regards

**Comment by flonkler** [comment]
Hi sgatrip,

I am currently re-writing the whole project switching from YOLOv2 to, well, something I came up with.

To answer your questions:
1. The dataset I have is very tiny, that's for sure. I will add more and more receipts to the dataset and maybe provide a link to download it in the future. For now I am just testing the concept, trying to overfit a small dataset to see what model structure and optimization technique is best for the future.

2. I use keras with tensorflow backend. You can look up more details in the code (optimization algorithm, etc.) and I also added a `config.json` file which includes neccessary hyperparameters like learning rate and batch size for example.

3. I will implement validation and visualization of the model's prediction in the next commits.

Have a nice day!
